src/level.rs

```diff
@@ -103,7 +102,6 @@ fn initialize_level_grid(world: &mut World, grid_config: Grid) {
                 transform.rotate_local(Vector3::new(0.0, 1.0, 0.0), Deg(wall_rotation as f32));
 
                 let mesh_path = format!("meshes/{}", wall_type);
-                let texture_path = format!("textures/{}", wall_type);
```

Comment:
stays!

Reply:
but why?

Reply:
comment it if you want, 

because ill forget that textures belong to another folder otherwise
xD